modules/project-comments/client/services/comment.model.service.js

```diff
@@ -284,7 +284,7 @@ angular.module('comment').factory('CommentModel', ['$q', 'ModelBase', 'moment',
           a.push(row.commentId);
           a.push(commentFormat(row.comment));
           a.push(commentFormat(row.proponentResponse));
-          a.push(row.showProponentResponse);
+          a.push(row.proponentResponse ? (row.showProponentResponse ? 'Published' : 'Yes') : 'No');
```

Comment:
Ohh I was wondering how to do this yesterday with the ternary operator. I ended up just using three ifs.